cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -533,13 +534,14 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             };
         }
     });
+    
     GradingEditor = AbstractEditor.extend({
         templateName: 'grading-editor',
         className: 'edit-settings-grading',
 
         afterRender: function () {
             AbstractEditor.prototype.afterRender.call(this);
-            this.setValue(this.model.get('format'));
+            this.setValue(this.model.get('format') || 'notgraded');
```

Comment:
Checkout `grading_editor.underscore`. The default grading type should be not graded right? There was a bug where `this.model.get('format')` can be undefined resulting in the select box not being properly set. I assumed that this case was semantically equivalent to "not graded". Can you confirm?